app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
```

Comment:
You do not need to import these classes, because they are subclasses of the VaccineError superclass. You can write: from app.errors import VaccineError, NotWearingMaskError and delete the next line.